libnopegl/src/api.c

```diff
@@ -98,7 +98,7 @@ static int get_default_platform(void)
 #elif defined(TARGET_WINDOWS)
     return NGL_PLATFORM_WINDOWS;
 #else
-    return NGL_ERROR_UNSUPPORTED;
+    ngli_assert(0);
```

Comment:
Can you put a static assert so that it doesn't even compile?

Reply:
Done.